README.md

```diff
@@ -15,6 +15,35 @@ See https://github.com/TACC-Cloud/geoapi which is an associated restful API.
 
 `react/` has the react client
 
+
+#### Environments
+
+Environments are handled by vite via `vite.config.ts` and the `.env` files.
+
+
+The vite commands defined in `package.json` with the `--mode` prefix determines which environment we are running/building in.
+
+
+Then, for local development, the `BACKEND` specified in `.env` will allow testing different backends.
```

Comment:
should we split this BACKEND variable into two different ones:
* GEOAPI_BACKEND
* DESIGNSAFE_PORTAL

maybe for the short term this suggestion isn't so important. thoughts?  i can just make a follow on ticket.